src/Jobs/NodeChannelSuscribeJob.cs

```diff
@@ -0,0 +1,187 @@
+    private readonly INodeRepository _nodeRepository;
+    private readonly IChannelRepository _channelRepository;
+    private readonly ISchedulerFactory _schedulerFactory;
+    
+    public NodeChannelSuscribeJob(ILogger<NodeChannelSuscribeJob> logger, ILightningService lightningService, INodeRepository nodeRepository, IChannelRepository channelRepository, ISchedulerFactory schedulerFactory)
+    {
+        _logger = logger;
+        _lightningService = lightningService;
+        _nodeRepository = nodeRepository;
+        _channelRepository = channelRepository;
+        _schedulerFactory = schedulerFactory;
+    }
+        
+    public async Task Execute(IJobExecutionContext context)
+    {
+        var data = context.JobDetail.JobDataMap;
+        var nodeId = data.GetInt("nodeId");
+        _logger.LogInformation("Starting {JobName} for node {nodeId}... ", nameof(NodeChannelSuscribeJob), nodeId);
+        try
+        {
+            var node = await _nodeRepository.GetById(nodeId);
+            
+            if (node == null)
+            {
+                _logger.LogInformation("The node: {@Node} is no longer ready to be supported quartz jobs", node);
+                return;
+            }
+
+            var client = LightningService.CreateLightningClient(node.Endpoint);
+            var result = client.Execute(x => x.SubscribeChannelEvents(new ChannelEventSubscription(), 
+                new Metadata {
+                    {"macaroon", node.ChannelAdminMacaroon}
+                }, null, default));
+
+            while (await result.ResponseStream.MoveNext())
+            {
+                node = await _nodeRepository.GetById(nodeId);
+            
+                if (node == null)
+                {
+                    _logger.LogInformation("The node: {@Node} is no longer ready to be supported quartz jobs", node);
+                    return;
+                }
+                
+                try {
+                    var channelEventUpdate = result.ResponseStream.Current;
+
+                    if (channelEventUpdate.OpenChannel != null)
+                    {
+                        if (String.IsNullOrEmpty(channelEventUpdate.OpenChannel.CloseAddress))
+                        {
+                            throw new Exception("Close address is empty");
+                        }
+                        var channelOpened = channelEventUpdate.OpenChannel;
+                        var fundingTxAndIndex = channelOpened.ChannelPoint.Split(":");
+                        var channel = new Channel()
+                        {
+                            ChanId = channelOpened.ChanId,
+                            SatsAmount = channelOpened.Capacity,
+                            Status = Channel.ChannelStatus.Open,
+                            IsAutomatedLiquidityEnabled = false,
+                            BtcCloseAddress = channelOpened.CloseAddress,
+                            FundingTx = fundingTxAndIndex[0],
```

Comment:
You can use fundingTxAndIndex.First() probably

Reply:
Should be the same right? Is that way more secure or better?